book/src/parachains/picasso/picasso-tokenomics.md

```diff
@@ -0,0 +1,59 @@
+# PICA Tokenomics
+
+
+## PICA use cases
+
+PICA is fundamental to governance, user dynamics, and network usage on the Picasso parachain. 
+
+
+### Gas fees
+
+Transaction fees (or gas) are payable in PICA.
+
+
+### Governance
+
+PICA will be used in voting to decide on key governance decisions, including: 
+
+
+
+* Pallet onboarding: which pallets can be incorporated on the Picasso parachain
+* Pallet graduation: which pallets on Picasso will be able to “graduate” onto the Composable parachain
+* Ecosystem growth: directing Treasury towards different initiatives in the ecosystem such as grants and incentivizing usage of the initial decentralized apps (dApps) on the network.
+* Public goods infrastructure: powering infrastructure providers, block explorers, wallets, and other key ecosystem partners.
+
+
+### Collator staking
+
+Collators for the Picasso parachain are required to put down a stake to produce blocks on the Picasso parachain. In doing so, collators will earn PICA through transaction fees.
+
+
+### Oracle staking
+
+Oracle operators are required to put down stakes to provide price feeds. They will be rewarded/ slashed according to the accuracy of the data they provide.
+
+
+### Liquidity provisioning
+
+Users can provide liquidity for PICA-USDC, PICA-KSM pairs on the Pablo DEX for yield. 
+
+
+## Total supply and genesis token distribution
+
+The total supply of PICA is 10 billion tokens. Token distribution is as the following:
```

Comment:
Please change this line to "The total supply of PICA is 10 billion tokens. PICA token distribution is intended to be as follows:"